test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -19,7 +24,12 @@ Describe "New-PSSession basic test" -Tag @("CI") {
 Describe "Basic Auth over HTTP not allowed on Unix" -Tag @("CI") {
     It "New-PSSession should throw when specifying Basic Auth over HTTP on Unix" -skip:($IsWindows) {
         $platformInfo = Get-PlatformInfo
-        if (($platformInfo -eq "alpine") -or ($platformInfo -eq "raspbian")) {
+        if (
+            ($platformInfo.Platform -eq "alpine") -or
+            ($platformInfo.Platform -eq "raspbian") -or
```

Comment:
Can you continue to use `-match "alpine|raspbian"` here?

Reply:
Fixed